src/components/graph-view.js

```diff
@@ -815,9 +825,6 @@ class GraphView extends Component {
       <div
           className='viewWrapper'
           tabIndex={0}
-          onClick={() => {
```

Comment:
Before someone asks, the click is handled in `handleSvgClicked` so this was a duplicate